src/algebra/module/basic.lean

```diff
@@ -371,11 +371,11 @@ end add_comm_group
 
 namespace add_monoid_hom
 
-lemma map_nat_module_smul [add_comm_monoid M] [add_comm_monoid M₂]
+lemma map_nat_module_smul [add_monoid M] [add_monoid M₂]
   (f : M →+ M₂) (x : ℕ) (a : M) : f (x • a) = x • f a :=
 f.map_nsmul a x
```

Comment:
This lemma can probably just be removed, although I guess that's out of scope for this PR